dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -885,6 +886,15 @@ protected String getCommitID(String repositoryId, Version version) {
         return null;
     }
 
+    public String getEmail(GHMyself myself) throws IOException {
+        for (GHEmail email: myself.getEmails2()) {
```

Comment:
Does `getEmails2` always return a collection? Take a look at code to be sure. Concerned about possible NPE.

Reply:
From the documentation, getEmails2 always return a collection. The code throws a IOException, but I do not see a NPE being thrown in the code, or in the methods that it uses. This documentation says the method returns 'always non-null': https://github-api.kohsuke.org/apidocs/org/kohsuke/github/GHMyself.html#getEmails2()